### PR TITLE
Security template updates

### PR DESCRIPTION
**Pull request by alexisgrow**

### CODE REVIEW

**Review by oshi97 — COMMENTED** [review]
Looks good to me! Only thing I'm not sure about is if whether it's necessary to add the SafeString method but I'm not really an expert on code cleanliness

File: src/ui/components/icons/iconcomponent.js [hunk 1]
```diff
@@ -80,7 +54,8 @@ export default class IconComponent extends Component {
    */
   setState (data) {
     return super.setState(Object.assign(data, {
-      image: this.image,
+      iconUrl: this.iconUrl,
+      iconName: this.iconName ? this.iconName : 'default',
```
Comment (tmeyer2115) [inline src/ui/components/icons/iconcomponent.js]: Given the default on line 19, I think we can skip the defaulting behavior here.

**Review by tmeyer2115 — COMMENTED** [review]
Looks great! Are there any stray references to customIcon in the README? If so, we should remove those.

File: src/ui/components/icons/iconcomponent.js (hunk 1 above)
Comment (alexisgrow) [reply src/ui/components/icons/iconcomponent.js]: Oh true, updating

**Review by tmeyer2115 — APPROVED** [review]

**Pull request merged**